meetings/2019-10-29.md

```diff
@@ -0,0 +1,95 @@
+# Code Curious Elixir Meeting 2019-10-29
+
+## Attendees
+
+- Jens (moderating)
+- Nicolo
+- Markus
+- Max
+- Tobi
+- Daniel
```

Comment:
Should be Gui (@ggpasqualino), not Daniel

Reply:
Haha, oh my god 🙈 So sorry @ggpasqualino 🙇 (I actually don't even know why I came up with that in the first place...)